kratos/python/add_model_to_python.cpp

```diff
@@ -26,19 +26,18 @@ namespace Python {
 void  AddModelToPython(pybind11::module& m)
 {
     namespace py = pybind11;
-    py::class_<Model >(m,"Model")
-    .def(py::init<>())
-    .def("Reset", &Model::Reset)
-    .def("CreateModelPart", [&](Model& self, const std::string& Name){return &self.CreateModelPart(Name);}, py::return_value_policy::reference_internal )
```

Comment:
Why do you change this?
I think the current one is what most of us do

Reply:
It has been automatically changed by the editor. In any case Google style places the ampersand together with the variable name.